src/EnergyPlus/UnitVentilator.cc

```diff
@@ -2224,7 +2224,16 @@ namespace UnitVentilator {
                 }
             } else {
                 CheckZoneSizing(state, state.dataUnitVentilators->cMO_UnitVentilator, state.dataUnitVentilators->UnitVent(UnitVentNum).Name);
-                OutAirVolFlowDes = state.dataUnitVentilators->UnitVent(UnitVentNum).MaxAirVolFlow;
```

Comment:
If OA Control = FixedAmount match the min OA for reporting purposes.

Reply:
So with this change, the unit ventilator will now respect the FinalZoneSizing MinOA value, and only if the OA control type is fixed OA control.  Interesting.

Reply:
Correct. In getInput the model uses the minimum flow for FixedAmount control. But when autosizing these air vol flows, the sizing routine forgot about that requirement.

            if (state.dataUnitVentilators->UnitVent(UnitVentNum).OAControlType == state.dataUnitVentilators->FixedOAControl) {
                state.dataUnitVentilators->UnitVent(UnitVentNum).OutAirVolFlow = state.dataUnitVentilators->UnitVent(UnitVentNum).MinOutAirVolFlow;
                state.dataUnitVentilators->UnitVent(UnitVentNum).MaxOASchedName = state.dataUnitVentilators->UnitVent(UnitVentNum).MinOASchedName;
                state.dataUnitVentilators->UnitVent(UnitVentNum).MaxOASchedPtr =
                    GetScheduleIndex(state, state.dataUnitVentilators->UnitVent(UnitVentNum).MinOASchedName);
            }